usaspending_api/transactions/delta_models/summary_state_view.py

```diff
@@ -87,7 +90,8 @@
                 0
             ) AS NUMERIC(23, 2)
         ) AS face_value_loan_guarantee,
-        COUNT(*) AS counts
+        COUNT(*) AS counts,
+        0 AS total_outlays  -- Default value for new column
```

Comment:
Might make sense to make this a `NULL` instead of `0`. We want to make sure that we can distinguish between the case that a state actually has $0 in funds vs. if this field isn't populated by the subsequent queries for some reason that we can d

Reply:
I have made the correction.